### ISSUE TITLE
Reader: Open Settings popover by default

### ISSUE THREAD

**Issue by jancavan** [Automattic/wp-calypso#21622]
In the Site Stream, after clicking the Follow button, the Settings popover should be open by default. This aids in better discoverability of the Post Notifications feature.

Perhaps we can do this as an AB test?

**Comment by blowery** [comment]
> Perhaps we can do this as an AB test?

We could indeed!

> **In the Site Stream**, after clicking the Follow button, the Settings popover should be open by default. This aids in better discoverability of the Post Notifications feature.

The site stream doesn't generate that many follows compared to other places in the UI, especially search results and tag streams. Do you think there's a way we could make this work there too?

**Comment by jancavan** [comment]
> Do you think there's a way we could make this work there too?

Good call! Yes, those two are good places to add it. We can add it to the cards in Post Results and Search Recommendations. Quick mockup:

![settings-in-card](https://user-images.githubusercontent.com/4924246/35131172-603bfde8-fc79-11e7-98b7-04ba847caa05.png)

In Search, it might end up being too much, especially if they follow one site after another. So what if we display it by default once, but we don't show it by default again for the next thing they follow? I think we'll have to test it in order to get a feel for it.

Sidenote: I think we should remove global notifications for Follow/Unfollow actions in the site stream. They're really obtrusive:

![screenshot 2018-01-18 17 55 54](https://user-images.githubusercontent.com/4924246/35131085-f3dacc24-fc78-11e7-950d-497441618c06.png)

**Comment by jancavan** [comment]
> Sidenote: I think we should remove global notifications for Follow/Unfollow actions in the site stream. They're really obtrusive:

Created an issue for this: https://github.com/Automattic/wp-calypso/issues/21668

Added it to the Post Notifications milestone because I think it affects the Settings button making it less discoverable.

**Comment by blowery** [comment]
Thanks @jancavan!

Couple more questions on this: Would we prefer to wait until the network response comes back with success to show the settings popover? If we want to show it immediately, what should we do if the follow fails?

**Comment by jancavan** [comment]
> Would we prefer to wait until the network response comes back with success to show the settings popover? If we want to show it immediately, what should we do if the follow fails?

Is it possible to show it only if the follow has been successful? Doesn't the Settings button itself appear only after a successful follow?

**Comment by blowery** [comment]
![follow-unfollow-cog](https://user-images.githubusercontent.com/14350/35283354-08ae64a2-0026-11e8-99f8-e33e3bdb8a3a.gif)

Looks like we're showing it optimistically currently.

**Comment by jancavan** [comment]
> Looks like we're showing it optimistically currently.

Can we use the same approach with the popover? So if you click follow, both Settings and popover get displayed, but if the follow is unsuccessful, the button just reverts to its original state and we show the current error message?

**Comment by blowery** [comment]
Yup, sounds like a plan!

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

As there has been no recent activity on this issue it is being closed with the label `Closed - Won't Fix`.
If this issue should remain open, please feel free to re-open this bug.
